test/check-application

```diff
@@ -704,7 +704,7 @@ class TestApplication(testlib.MachineCase):
         b.set_checked("#run-image-dialog-tty", True)
 
         # Set up command line
-        b.set_input_text('#run-image-dialog-command', "sh -c 'for i in $(seq 20); do sleep 1; echo $i; done'")
+        b.set_input_text('#run-image-dialog-command', "sh -c 'for i in $(seq 20); do sleep 1; echo $i; done; sleep infinity'")
```

Comment:
That's a really curious command now :-) Do  we already  have a different one that just does `sleep infinity`, and we need that for disambiguation?

Reply:
no really. I use this command to test logs, thats why I have there that for loop. It however cannot exit. I could just do the loop up to 1000 or so and that would do the trick, but thought why not just keeping it alive explicitly.